fieldservice/views/fsm_stage.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Field Service Stage -->
+    <record id="fsm_stage_tree_view" model="ir.ui.view">
+        <field name="name">Stages</field>
+        <field name="model">fsm.stage</field>
+        <field name="arch" type="xml">
+            <tree string="Stages">
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="fsm_stage_form_view" model="ir.ui.view">
+        <field name="name">Stage</field>
+        <field name="model">fsm.stage</field>
+        <field name="arch" type="xml">
+            <form string="Stage">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name"/>
+                            <field name="sequence"/>
+                        </group>
+                        <group>
+                            <field name="fold"/>
+                        </group>
```

Comment:
It would be good to include Teams in the FSM base module, and allow for different Stages for each team:
my "Installations" team might have a different workflow from my "Repairs" teams.

Reply:
See #5